packages/docusaurus-plugin-content-blog/src/__tests__/pluginOptionSchema.test.ts

```diff
@@ -47,7 +47,10 @@ test('should accept valid user options', async () => {
     ],
   };
   const {value, error} = await PluginOptionSchema.validate(userOptions);
-  expect(value).toEqual(userOptions);
+  expect(value).toEqual({
+    ...userOptions,
+    feedOptions: {type: ['rss', 'atom']},
+  });
```

Comment:
yes

Reply:
I believe I tried what you suggested when trying to resolve the issue with this unit test. However, it won't pass the test unless we add `feedOptions: {type: ['rss', 'atom']}` here

Reply:
why exactly this test does not pass? 

I can't tell you if I can't see the code, but what I'm sure of is that we 100% want to put default options in the `DEFAULT_OPTIONS `  constant

Reply:
I've now added `type: 'all'` into `DEFAULT_OPTIONS.feedOptions`, so I think it would make sense that it would convert 'all' into the array (`['rss', 'atom']`) right?

Reply:
@cindygu4 I'd rather have `type: ['rss', 'atom']` and allow the validation to work with arrays syntax, so that the default options are not "transformed"

Reply:
Thanks for the suggestion @slorber! @vikhramt007 and I have implemented the feature using the `type: ['rss', 'atom']` as the default and allowed the validation to work with arrays. It now passes all of the unit tests :)